runatlantis.io/.vuepress/styles/index.scss

```diff
@@ -1,41 +1,243 @@
+// https://v2.vuepress.vuejs.org/reference/default-theme/styles.html#style-file
+
+// colors
+$textColor: var(--c-text);
+$borderColor: var(--c-border);
+$buttonTextColor: var(--c-badge-danger-text);
+$buttonColor: var(--c-brand);
+$buttonHoverColor: var(--c-brand-light);
+$darkBackground: var(--c-bg-light);
+$darkBackgroundBorder: var(--c-border-dark);
```

Comment:
When using var color, we can apply dark mode easily. All vars are defined in the following.

https://v2.vuepress.vuejs.org/reference/default-theme/styles.html#style-file